src/asset-services.ts

```diff
@@ -0,0 +1,123 @@
+function ensureAbsoluteURLs(baseURL: string, entrypoints: string[]) {
+  const regExpIsAbsoluteURL = new RegExp("^(?:[a-z]+:)?//", "i");
+  return entrypoints.map(function (entrypoint) {
+    if (!regExpIsAbsoluteURL.test(entrypoint)) {
+      return baseURL + entrypoint;
+    }
+    return entrypoint;
+  });
+}
+
+function addRef(document: Document, entrypoint: string) {
+  const pattern = /\.([0-9a-z]+)(?=[?#])|(\.)(?:[\w]+)$/gim;
+  switch (entrypoint.match(pattern)![0]) {
+    case ".css":
+      let link = document.createElement("link");
+      link.rel = "stylesheet";
+      link.href = entrypoint;
+      (link as any).async = false; // TODO: it seems to be superfluous, remove
+      // TODO: DE-666 - render this element near the script element where load is invoked
+      document.head.appendChild(link);
+      break;
+    case ".js":
+      let script = document.createElement("script");
+      script.src = entrypoint;
+      script.async = false;
+      // TODO: DE-666 - render this element near the script element where load is invoked
+      document.body.appendChild(script);
+      break;
+    default:
+      break;
```

Comment:
Pensamos hacer algo con esto a futuro?

Reply:
No, solo estaba resolviendo el error generado por [la regla _default-case_ de ESLint](https://eslint.org/docs/rules/default-case).

Que opinás de que haga lo siguiente?

```suggestion
    default:
      // do nothing
      break;
```